File: Simplace/SimplaceReferenceETPriestleyTaylor/src/java/SimplaceReferenceETPriestleyTaylor/ReferenceETPriestleyTaylor_Exogenous.java
import  java.io.*;
import  java.util.*;
import java.time.LocalDateTime;

public class ReferenceETPriestleyTaylor_Exogenous
{
    private double iTMin;
    private double iNetRadiation;
    private double iTMax;
    
    public ReferenceETPriestleyTaylor_Exogenous() { }
    
    public ReferenceETPriestleyTaylor_Exogenous(ReferenceETPriestleyTaylor_Exogenous toCopy, boolean copyAll) // copy constructor 
    {
        if (copyAll)
        {
            this.iTMin = toCopy.getiTMin();
            this.iNetRadiation = toCopy.getiNetRadiation();
            this.iTMax = toCopy.getiTMax();
        }
    }
    public double getiTMin()
    { return iTMin; }

    public void setiTMin(double _iTMin)
    { this.iTMin= _iTMin; } 
    
    public double getiNetRadiation()
    { return iNetRadiation; }

    public void setiNetRadiation(double _iNetRadiation)
    { this.iNetRadiation= _iNetRadiation; } 
    
    public double getiTMax()
    { return iTMax; }

    public void setiTMax(double _iTMax)
    { this.iTMax= _iTMax; } 
    
}

File: Simplace/SimplaceReferenceETPriestleyTaylor/src/cs/SimplaceReferenceETPriestleyTaylor/ReferenceETPriestleyTaylor_Exogenous.cs
using System;
using System.Collections.Generic;

public class ReferenceETPriestleyTaylor_Exogenous 
{
    private double _iTMin;
    private double _iNetRadiation;
    private double _iTMax;
    
    /// <summary>
    /// Constructor of the ReferenceETPriestleyTaylor_Exogenous component")
    /// </summary>  
    public ReferenceETPriestleyTaylor_Exogenous() { }
    
    
    public ReferenceETPriestleyTaylor_Exogenous(ReferenceETPriestleyTaylor_Exogenous toCopy, bool copyAll) // copy constructor 
    {
        if (copyAll)
        {
    
            iTMin = toCopy.iTMin;
            iNetRadiation = toCopy.iNetRadiation;
            iTMax = toCopy.iTMax;
        }
    }
    public double iTMin
    {
        get { return this._iTMin; }
        set { this._iTMin= value; } 
    }
    public double iNetRadiation
    {
        get { return this._iNetRadiation; }
        set { this._iNetRadiation= value; } 
    }
    public double iTMax
    {
        get { return this._iTMax; }
        set { this._iTMax= value; } 
    }
}